SNUTT-2022/SNUTT/Models/User.swift

```diff
@@ -2,13 +2,35 @@
 //  User.swift
 //  SNUTT
 //
-//  Created by Jinsup Keum on 2022/06/25.
+//  Created by 최유림 on 2022/08/18.
 //
 
-import SwiftUI
+import Foundation
 
-class User: ObservableObject {
+struct User {
     var localId: String?
     var fbName: String?
     var email: String?
 }
+
+extension User {
+    init(from data: UserDto) {
+        localId = data.local_id
+        fbName = data.fb_name
+        email = data.email
+    }
+}
```

Comment:
사소한거긴 한데 기존에는 `data` 대신 `dto`라는 변수명을 썼었어요.

```suggestion
extension User {
    init(from dto: UserDto) {
        localId = dto.local_id
        fbName = dto.fb_name
        email = dto.email
    }
}
```